### ISSUE TITLE
Accessibility: Plugin Title should be h2

### ISSUE THREAD

**Issue by joedolson** [aspirepress/AspireExplorer#34]
Within the context of the current hierarchy, the plugin title should be an `h2`. If all of the sections are `h3`, the parent heading needs to be one level higher.

**Comment by joedolson** [comment]
Making this templatable might be better; H2 is only appropriate in the FAIR theme because this is being embedded inside the single template, but that should probably be modified, in which case this would actually need to be an h1, and all other headings should move up one level.
